src/ngraph/frontend/onnx_import/utils/reduction.cpp

```diff
@@ -32,17 +32,23 @@ namespace ngraph
             {
                 AxisSet get_reduction_axes(const Node& node)
                 {
-                    auto reduction_axes =
-                        node.get_attribute_value<std::vector<std::int64_t>>("axes", {});
-                    std::vector<std::size_t> normalized_axes =
+                    const auto data_ps = node.get_ng_inputs().at(0)->get_output_partial_shape(0);
+                    NGRAPH_CHECK(
```

Comment:
This check should not be needed after @mbencer's changes to `normalize_axes` are merged.